packages/wallets/src/components/WalletsResetMT5Password/WalletSuccessResetMT5Password.tsx

```diff
@@ -3,35 +3,28 @@ import { Trans } from 'react-i18next';
 import { DerivLightIcMt5PasswordUpdatedIcon, DerivLightMt5SuccessPasswordResetIcon } from '@deriv/quill-icons';
 import useDevice from '../../hooks/useDevice';
 import { ModalStepWrapper, WalletButton } from '../Base';
-import { useModal } from '../ModalProvider';
 import { WalletsActionScreen } from '../WalletsActionScreen';
 
 type WalletSuccessResetMT5PasswordProps = {
     isInvestorPassword?: boolean;
-    onClickSuccess?: () => void;
+    onClick: () => void;
     title: string;
 };
 
 const WalletSuccessResetMT5Password: FC<WalletSuccessResetMT5PasswordProps> = ({
     isInvestorPassword = false,
-    onClickSuccess,
+    onClick,
     title,
 }) => {
-    const { hide } = useModal();
     const { isMobile } = useDevice();
 
-    const handleSuccess = useCallback(() => {
-        onClickSuccess?.();
-        hide();
-    }, [onClickSuccess, hide]);
-
```

Comment:
why is passing the `hide` inside the `handleSuccess` fn the issue?

Reply:
question is _why_ ?

`onClickSuccess?.();` is not being used or doing anything here